sui/its.js

```diff
@@ -469,6 +469,10 @@ async function linkCoin(keypair, client, config, contracts, args, options) {
     const { Gateway } = AxelarGateway.objects;
     const [symbol, name, decimals, destinationChain, destinationAddress] = args;
 
+    // Token manager type
+    const tokenManagerTypes = ['lock_unlock', 'mint_burn'];
+    const tokenManager = options.tokenManagerMode ? options.tokenManagerMode : tokenManagerTypes[0];
```

Comment:
```suggestion
    const tokenManagerType = options.tokenManagerMode;
```
why is this needed? This option will always be populated correctly